Protocol.md

```diff
@@ -318,16 +318,14 @@ Frame Contents
     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
     |                           Stream ID                           |
     +-----------+-+-+---------------+-------------------------------+
-    |Frame Type |0|M|      Flags    |
+    |Frame Type |0|0|      Flags    |
     +-----------+-+-+---------------+-------------------------------+
     |                          Error Code                           |
     +---------------------------------------------------------------+
-                        Metadata & Error Data
+                               Error Data
 ```
 
 * __Frame Type__: (6 bits = max value 63) 0x0B
-* __Flags__: (10 bits)
-     * (__M__)etadata: Metadata present
 * __Error Code__: (32 bits = max value 2^31-1 = 2,147,483,647) Type of Error.
      * See list of valid Error Codes below.
 * __Error Data__: includes Payload describing error information. Error Data SHOULD be a UTF-8 encoded string. The string MUST NOT be null terminated.
```

Comment:
Specifically my question is how to handle the behaviour that thrift adopts where idl defined exceptions are strongly typed.

We remove the Metadata with agreed Mimetype and replace it with a String.  Now to encode something like a thrift error we need to map the idl error codes to the > 0x0300 error code.

More likely most error handling will become substring matching of known error strings